pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -251,6 +251,7 @@ type TaskRunStatusFields struct {
 	// +listType=atomic
 	Steps []StepState `json:"steps,omitempty"`
 
+	// Deprecated. This field will be removed in a future release.
```

Comment:
I don't know if we'll remove this field from v1beta1 TaskRuns, rather than just waiting until we no longer support v1beta1. I'd just keep it as "deprecated".